preparedstatement/src/main/java/software/amazon/athena/preparedstatement/ClientBuilder.java

```diff
@@ -0,0 +1,13 @@
+package software.amazon.athena.preparedstatement;
+
+import software.amazon.awssdk.services.athena.AthenaClient;
+import software.amazon.cloudformation.LambdaWrapper;
+
+public class ClientBuilder {
+
+  public static AthenaClient getClient() {
+    return AthenaClient.builder()
+              .httpClient(LambdaWrapper.HTTP_CLIENT)
```

Comment:
Do we need to add any retry strategy or timeout configurations while creating the client?

Reply:
I don't think any non default client configuration provides much benefit, these handlers are called by the CFN service which has its own service limits exposed to customers, and each CFN resource can have its own retry/timeout configuration when we register the resource type version with CFN.